src/main/scala/com/elarib/PartialSbtPlugin.scala

```diff
@@ -4,11 +4,15 @@ import com.elarib.model.{ChangeGetter, PartialSbParser}
 import org.apache.logging.log4j.LogManager
 import sbt.Keys._
 import sbt.internal.BuildDependencies.DependencyMap
-import sbt.{AutoPlugin, Command, Def, _}
+import sbt._
+
+object BuildKeys {
+  val partialSbtExcludeFiles = sbt.settingKey[Seq[sbt.File]]("Files that should be excluded from analysis.")
```

Comment:
Here and in other places it should probably be `Excluded`. It's a property, not a command.